nipy/algorithms/statistics/onesample.py

```diff
@@ -132,6 +135,6 @@ def estimate_varatio(Y, sd, df=None, niter=10):
     value['random'] = sigma2
 
     if squeeze:
-        for key in value.keys():
+        for key in list(value):
             value[key] = np.squeeze(value[key])
```

Comment:
Hmm.  Python 3 doesn't allow you to modify the keys when iterating over the dictionary keys, but it does allow you to modify the values, so I think you're right, list isn't necessary.

```
In [11]: a = dict(one=1, two=2)

In [12]: for key in a:
   ....:     if key == 'two': a['three'] = 3
   ....:     
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-12-6b63c5f5ae33> in <module>()
----> 1 for key in a:
      2     if key == 'two': a['three'] = 3
      3 

RuntimeError: dictionary changed size during iteration

In [13]: for key in a:
    if key == 'two': a['one'] = 3
   ....:     

In [14]: 
```